Pyrado/pyrado/sampling/step_sequence.py

```diff
@@ -343,6 +343,12 @@ def __init__(
         else:
             self._rollout_bounds = None
 
+    def __repr__(self) -> str:
+        return f"StepSequence[rewards={self.rewards}, observations={self.observations}, actions={self.actions}]"
```

Comment:
Please use line breaks, of even better tabulate.tabulate() here, since __repr__ sould be human readable